lambdas/cfn-ssm-param-macro/tests/events/string.json

```diff
@@ -0,0 +1,8 @@
+{
+  "fragment": {},
+  "requestId": "foo",
+  "params": {
+    "Type": "String",
+    "Name": "test-param"
```

Comment:
is there a test where the "Name" key is present however the value does not exist in the SSM?

Reply:
Just added this